### PR TITLE
Add css class based on if results were returned from a find

### PR DESCRIPTION
**Pull request by zoo1**
enhancement for #761

### CODE REVIEW

**Comment by simurai** [comment]
Thanks for improving this. 👍 

We probably should add some specs to make sure the classes don't get removed. I don't know too much about adding specs.. but does adding

```coffee
describe "when no results are found", ->
  it "adds a .has-no-results class", ->
    findView.findEditor.setText 'notinthefile'
    atom.commands.dispatch(findView.findEditor.element, 'find-and-replace:confirm')
    expect(findView).toHaveClass 'has-no-results'

describe "when results are found", ->
  it "adds a .has-results class", ->
    findView.findEditor.setText 'items'
    atom.commands.dispatch(findView.findEditor.element, 'find-and-replace:confirm')
    expect(findView).toHaveClass 'has-results'
```

maybe [here](https://github.com/atom/find-and-replace/blob/master/spec/find-view-spec.coffee#L351) make sense?

**Comment by simurai** [comment]
Another thing, the classes should get removed when either condition is not met.

search

Currently the `has-no-results` and `has-results` stay until Atom gets reloaded.

File: lib/find-view.coffee [hunk 1]
```diff
@@ -307,6 +307,7 @@ class FindView extends View
     if @model.getFindOptions().findPattern
       results = @markers.length
       resultsStr = if results then _.pluralize(results, 'result') else 'No results'
+      @.removeClass('has-results has-no-results')
```
Comment (50Wliu) [inline lib/find-view.coffee]: Use `this` instead of `@` when it would otherwise be standalone.

**Comment by simurai** [comment]
Ok, this seems ready to 🚢 

Thanks @zoo1

**Pull request merged**